roles/a_train/templates/a-train.toml.j2

```diff
@@ -0,0 +1,13 @@
+# a-train.toml
+[autoscan]
+# Replace the URL with your Autoscan URL.
+url = "http://autoscan:3030"
```

Comment:
Suggest using vars here to account for potential overrides.
``url = "{{ 'http://' + autoscan_docker_networks_alias  + ':' + autoscan_web_port }}"``